src/openrct2-ui/windows/TitleEditor.cpp

```diff
@@ -836,7 +857,7 @@ static void window_title_editor_scrollpaint_saves(rct_window *w, rct_drawpixelin
 static void window_title_editor_scrollpaint_commands(rct_window *w, rct_drawpixelinfo *dpi)
 {
     sint32 position = -1;
-    if (_isSequencePlaying) {
+    if (_isSequencePlaying && (uint16)_selectedTitleSequence == title_get_current_sequence()) {
```

Comment:
The formatting in this file will need to be fixed. Could you open another PR for that once this one is merged?

Reply:
Yeah definitely.